### PR TITLE
Add chatwoot support system

### PR DESCRIPTION
**Pull request by xpayserver**
Hi I add support for chat support system that soon integrate into BTCPay.

### CODE REVIEW

**Comment by NicolasDorier** [comment]
wow that's cool. Will test it.

**Comment by NicolasDorier** [comment]
Why requiring an environment file? You can set your own parameters with a custom docker fragment instead.

File: docs/chatwoot.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+```bash
+CHATWOOT_HOST="chatwoot.xpayserver.com"
+BTCPAYGEN_ADDITIONAL_FRAGMENTS="$BTCPAYGEN_ADDITIONAL_FRAGMENTS;opt-add-chatwoot"
+. btcpay-setup.sh -i
+
+```
+4. Wait for BTPay to be online and then create the database for chatwoot
+
+```bash
+docker exec -ti chatwoot sh -c "export DISABLE_DATABASE_ENVIRONMENT_CHECK=1 && bundle exec rails db:reset"
```
Comment (NicolasDorier) [inline docs/chatwoot.md:29]: Can't this step be done automatically with `entrypoint`?

Comment (xpayserver) [reply docs/chatwoot.md:29]: no this only for create DB. Chatwoot no create db automatically..

**Comment by xpayserver** [comment]
> Why requiring an environment file? You can set your own parameters with a custom docker fragment instead. This is how we do everywhere else.

This require because there many many config item that user must define which otherwise need pass through all btcpayserver-docker system. User need set SMTP, inbound email, storage which easy 20+ config https://github.com/chatwoot/chatwoot/blob/develop/.env.example

This PR work now but I switch focus to alt service #355  as it seem better. I merge both as both useful for user choice

**Comment by NicolasDorier** [comment]
@xpayserver why not removing the env file and just documenting how to add a custom docker fragment to set all of those config?

**Comment by britttttk** [comment]
@xpayserver can you post the github link so we can see what chatwood is?

**Review by Kukks — APPROVED** [review]
LGTM

**Comment by uhlhosting** [comment]
Everyone want to look into erxes.io . I would see a way more easier integration, from paid or self hosted version managed via yarn, fast responsive development team.

**Pull request merged**